integration-tests/src/test/java/org/openmrs/module/fhir2/providers/r3/LocationFhirResourceProviderIntegrationTest.java

```diff
@@ -483,6 +486,31 @@ public void shouldSearchForExistingLocationsAsJson() throws Exception {
 		assertThat(entries, containsInRelativeOrder(hasResource(hasProperty("name", equalTo("Test location 6"))),
 		    hasResource(hasProperty("name", equalTo("Test location 8")))));
 		assertThat(entries, everyItem(hasResource(validResource())));
+		
+		// search by ancestors
+		response = get("/Location?below=" + LOCATION_ANCESTOR_TEST_UUID).accept(FhirMediaTypes.JSON).go();
```

Comment:
So when I changed this to "Location?partof:below=" it was getting picked up by the ResourceParam, but the "below" was being set to the ResourceType.  This seems incorrect?  Isn't the type of this resource a location?

![image](https://github.com/user-attachments/assets/e48642be-e829-4e22-80cc-e1af6b5ebe1a)